app/query/caller.go

```diff
@@ -253,12 +253,16 @@ func (c *Caller) SendQuery(q *Query) (*jsonrpc.RPCResponse, error) {
 		}
 	}
 
-	logFields := logrus.Fields{
-		"method":   q.Method(),
-		"params":   q.Params(),
-		"endpoint": c.endpoint,
-		"user_id":  c.userID,
-		"duration": c.Duration,
+	logFields := logrus.Fields{}
+	// don't log params for "sync_apply" method
+	if q.Method() != MethodSyncApply {
+		logFields = logrus.Fields{
+			"method":   q.Method(),
+			"params":   q.Params(),
```

Comment:
We only want to omit _JSON-RPC query params_, not all params. It would not make much sense to omit method name or duration for sync apply calls, for example.

In other words, only `q.Params()` needs to be skipped for `sync_apply` log call.

Reply:
Understood, my apologies.  If you keep this open I can get a fix up today.

Thank you for your patience.